apps/s_server.c

```diff
@@ -3340,6 +3341,10 @@ static int www_body(int s, int stype, int prot, unsigned char *context)
             }
             /* send the file */
 #ifndef OPENSSL_NO_KTLS
+            if (use_sendfile_for_req && !BIO_get_ktls_send(SSL_get_wbio(con))) {
+                BIO_printf(bio_err, "Warning: sendfile requested but KTLS is not available\n");
+                use_sendfile_for_req = 0;
+            }
             if (use_sendfile) {
```

Comment:
I suppose you also have to change this to `if (use_sendfile_for_req)`.

Reply:
Fixed.